integration-tests/tests/integration.rs

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2024 by IBM.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use kbs::admin::config::AdminConfig;
+use kbs::attestation::config::{AttestationConfig, AttestationServiceConfig};
+use kbs::config::HttpServerConfig;
+use kbs::config::KbsConfig;
+use kbs::policy_engine::PolicyEngineConfig;
+use kbs::token::AttestationTokenVerifierConfig;
+use kbs::ApiServer;
+
+use kbs::plugins::{
+    implementations::{resource::local_fs::LocalFsRepoDesc, RepositoryConfig},
+    PluginsConfig,
+};
+
+use attestation_service::{
+    config::Config,
+    rvps::{RvpsConfig, RvpsCrateConfig},
+    token::{ear_broker, simple, AttestationTokenConfig},
+};
+
+use anyhow::{bail, Result};
+use log::info;
+use openssl::pkey::PKey;
+use rstest::rstest;
+use serde_json::json;
+use serial_test::serial;
+use std::io::Write;
+use tempfile::{NamedTempFile, TempDir};
+
+const KBS_URL: &str = "http://127.0.0.1:8080";
+const SECRET_BYTES: &[u8; 8] = b"shhhhhhh";
+const SECRET_PATH: &str = "default/test/secret";
+const WAIT_TIME: u64 = 3000;
+
+const ALLOW_ALL_POLICY: &str = "
+    package policy
+    allow = true
+";
+
+const DENY_ALL_POLICY: &str = "
+    package policy
+    allow = false
+";
+
+#[allow(dead_code)]
```

Comment:
Why do we need this lint macro?

Reply:
The `Custom` variant is never instantiated. I could move the dead code thing just to that variant.